spec/lib/appsignal/event_formatter_spec.rb

```diff
@@ -2,6 +2,7 @@ class MockFormatter < Appsignal::EventFormatter
   attr_reader :body
 
   def initialize
+    super
```

Comment:
For anyone else checking this PR: `EventFormatter` does not have an initialiser, so this is a no-op.